airflow-comparison.md

```diff
@@ -0,0 +1,47 @@
+## Lineage
+
+Data lineage broadly refers to a collection of tools, practices and approaches which allow the tracking of the flow of data within a data pipeline. Data lineage tools maintain a record of data within its lifecycle, including its source and any transformations applied to it. The main purpose of data lineage is to provide a platform for ensuring data quality, as well as a way of troubleshooting any errors which arise within the data.
+
+OpenLineage is 'an open platform for collection and analysis of data lineage’. It is the defacto industry standard for lineage collection, and it will be the basis for our comparison of lineage capabilities between Airflow and VDK.
```

Comment:
Seems this section could be "optimized" by saying it's defacto the open standard and providing a link, maybe would be sufficient.